protocol/git/README.md

```diff
@@ -106,6 +106,7 @@ View a list of new commits. View changed files. Merge branch into master.
     git log origin/master..<branch-name>
     git diff --stat origin/master
     git checkout master
+    git pull
```

Comment:
Actually, thinking a bit more about it, you're right. It's not necessary. Since we rebased on `origin/master` above, we know we've got changes from `origin`, so we'll get those as part of the merge. It's funny to me though that that piece is *slightly* hidden in this workflow.